### PR TITLE
Add GemJoin5

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

File: src/DssDeploy.t.sol [hunk 1]
```diff
@@ -664,80 +664,101 @@ contract DssDeployTest is DssDeployTestBase {
...
+        assertEq(vat.gem("GNT", address(this)), 6 ether);
+
+        usdc.approve(address(usdcJoin), uint(-1));
+        assertEq(usdc.balanceOf(address(usdcJoin)), 0);
+        assertEq(vat.gem("USDC", address(this)), 0);
+        usdcJoin.join(address(this), 10 * 10 ** 6);
+        assertEq(usdc.balanceOf(address(usdcJoin)), 10 * 10 ** 6);
+        assertEq(vat.gem("USDC", address(this)), 10 ether);
+        usdcJoin.exit(address(this), 4 * 10 ** 6);
+        assertEq(usdc.balanceOf(address(usdcJoin)), 6 * 10 ** 6);
```
Comment (kmbarry1) [inline src/DssDeploy.t.sol:774]: For completeness, could add:
```
assertEq(usdc.balanceOf(address(this), 4 * 10 ** 6);
```

Comment (gbalabasquer) [reply src/DssDeploy.t.sol:774]: Sure! I'll add it for every token

Comment (gbalabasquer) [reply src/DssDeploy.t.sol:774]: Done

**Review by kmbarry1 — APPROVED** [review]
LGTM

**Pull request merged**